Sources/NIOFileSystem/String+FileSystem.swift

```diff
@@ -0,0 +1,56 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2024 Apple Inc. and the SwiftNIO project authors
```

Comment:
2025? Or 2024-2025?

Reply:
This one still needs addressing. It should be 2025.

Reply:
🤦🏼  done